docs/docs/howto/connectors/My-connector-requests.rst

```diff
@@ -0,0 +1,23 @@
+My Connector Requests
+=====================
+
+The **My Connector Requests** page can be accessed through the **Requests -> My Connector Requests** navigation bar. You can view all requests created by you here. This page also provides an audit log and a history of the connector requests created by you.
+
```

Comment:
Is this My or is it My/My Team's requests
I think you can view all the requests by your team right?

Reply:
yes, we can view all my team's requests